tests/e2e/examples/dev_auth_bypass.py

```
@@ -0,0 +1,33 @@
+Example implementation of development auth bypass.
+This would need to be added to the actual QueryWeaver codebase.
+"""
+
+# In api/auth/user_management.py, add this function:
+
+async def get_test_user_for_development():
+    """
+    Return a test user for development/testing environments only.
+    NEVER enable this in production!
+    """
+    if os.getenv("APP_ENV") != "development" or not os.getenv("ENABLE_TEST_AUTH"):
```

Ruff (0.12.2)

Undefined name `os`

(F821)